pkg/bgpv1/test/adverts_test.go

```diff
@@ -701,6 +701,32 @@ func Test_AdvertisedPathAttributes(t *testing.T) {
 				},
 			},
 		},
+		{
+			description: "advertise service IP with common community",
+			op:          "add",
+			lbService: &lbSrvConfig{
+				name:      "service-b",
+				ingressIP: "10.100.1.112",
+			},
+			advertiseAttributes: []v2alpha1.CiliumBGPPathAttributes{
+				{
+					SelectorType: v2alpha1.CiliumLoadBalancerIPPoolSelectorName,
+					Communities: &v2alpha1.BGPCommunities{
+						Common: []v2alpha1.BGPCommonCommunity{v2alpha1.BGPCommonCommunity("no-export")},
```

Comment:
What happens if user defines both - common 'no-export' and standard '65535:65281', I assume goBGP will advertise filter out duplicate or does it throw an error. Also, is goBGP handling the case where we remove one but keep another; end result is the same.

Reply:
Yeah, this is interesting - as communities is a list, it will advertise the same community twice. No error on GoBGP side and FRR seems to be happy as well.

For example, with:
```
      - advertisedPathAttributes:
        - communities:
            common:
            - no-export
            standard:
            - 65535:65281
```

The result will be:
```
$ cilium bgp routes advertised
VRouter   Peer        Prefix          NextHop     Age      Attrs
65001     172.0.0.1   10.244.0.0/24   172.0.0.2   36m15s   [{Origin: i} {AsPath: 65001} {Nexthop: 172.0.0.2} {Communities: no-export, no-export}]
```
![image](https://github.com/cilium/cilium/assets/15926980/ea9f4268-f15f-4d7e-a466-3ba6a89e0251)

But the same can be achieved right now like this:
```
      - advertisedPathAttributes:
        - communities:
            standard:
            - 65535:65281
            - 65535:65281
```

Reply:
I looked at FRR, it is not possible to configure same community twice. Even when same community is expressed in different route map - only single community is advertised. 

I think we should filter out duplicates at control plane layer and advertise single community.

Reply:
Added community deduplication logic to the BGP CP. FYI, I also renamed `common` to `wellKnown` to align with naming in RFCs.